File: src/main/java/com/gtladd/gtladditions/api/machine/GTLAddWorkableElectricParallelHatchMultipleRecipesMachine.java
package com.gtladd.gtladditions.api.machine;

import org.gtlcore.gtlcore.common.data.GTLRecipeModifiers;

import com.gregtechceu.gtceu.api.machine.IMachineBlockEntity;

public class GTLAddWorkableElectricParallelHatchMultipleRecipesMachine extends GTLAddWorkableElectricMultipleRecipesMachine {

    public GTLAddWorkableElectricParallelHatchMultipleRecipesMachine(IMachineBlockEntity holder, Object... args) {
        super(holder, args);
    }

    public int getMaxParallel() {
        return GTLRecipeModifiers.getHatchParallel(this);
    }
}

File: src/main/java/com/gtladd/gtladditions/api/machine/GTLAddWorkableElectricParallelHatchMultipleRecipesMachine.kt
package com.gtladd.gtladditions.api.machine

import com.gregtechceu.gtceu.api.machine.IMachineBlockEntity
import org.gtlcore.gtlcore.common.data.GTLRecipeModifiers

open class GTLAddWorkableElectricParallelHatchMultipleRecipesMachine(holder: IMachineBlockEntity, vararg args: Any?) :
    GTLAddWorkableElectricMultipleRecipesMachine(holder, *args) {
    override fun getMaxParallel(): Int {
        return GTLRecipeModifiers.getHatchParallel(this)
    }
}
